content/enterprise_influxdb/v1.9/administration/backup-and-restore.md

```diff
@@ -316,12 +316,12 @@ Restored from my-incremental-backup/ in 83.892591ms, transferred 588800 bytes
 
 ##### Restore from a metadata backup
 
-In this example, the `restore` command restores an metadata backup stored
+In this example, the `restore` command restores a metadata backup stored
```

Comment:
This needs a much bigger warning I think.

There's actually two ways to backup and restore metadata:

1) Generate a metadata-only backup, and do a full restore with it. This re-numbers all the shards, but is safe to do as it can only be done in an empty cluster.

2) Do a `-meta-only` restore. This overwrites the full metadata state, and should not be done across clusters, only with a backup from an existing cluster. It is for the very specific circumstance where there are bad meta nodes but the data nodes in the cluster are still good, and we want to overwrite the meta state to some recent state without touching the data nodes.

In general a full cluster restore will be much safer than trying to do a meta-only restore. Before documenting, we should perhaps rename this meta-only flag to something scarier - `meta-only-overwrite` maybe?

Reply:
@timhallinflux FYI - I actually hadn't known about the meta-only backups when I wrote the `-meta-only` restore feature. Honestly I'm not sure what the `-meta-only` backup flag is intended for, since you basically still have an empty cluster after a restore from a metadata backup.

Any ideas on a good way forward?

Reply:
I believe the challenge that was faced by the customer was about the users, permissions, CQs, etc.  and wanted to move them between clusters.

Reply:
Proposal: I think the new flag should be named `meta-only-overwrite-force`, and documented as above. For moving users etc between clusters the meta backup is the way to go. `meta-only-overwrite-force` is to revert damaged meta nodes in an existing cluster to a previous state.

If that sounds good I can make a code change for it.

Reply:
the disambiguation is good.

Reply:
let’s change _megastore_ to _metadata_ thorough this doc, and in addition to clarifying the two ways to back up and restore metadata in this section, add the context under “Perform a metadata only backup” in line 179

Reply:
Here's the master PR, 1.9 PR coming after that is merged: [fix-meta-only](https://github.com/influxdata/plutonium/pull/3744)

Reply:
1.9 PR: https://github.com/influxdata/plutonium/pull/3745